app/_src/mesh/features/meshglobalratelimit.md

```diff
@@ -0,0 +1,663 @@
+{% navtab Kubernetes %}
+
+```yaml
+apiVersion: kuma.io/v1alpha1
+kind: ExternalService
+mesh: default
+metadata:
+  name: httpbin
+spec:
+  tags:
+    kuma.io/service: httpbin
+    kuma.io/protocol: http 
+  networking:
+    address: httpbin.org:443
+    tls:
+      enabled: true
+```
+
+{% endnavtab %}
+{% navtab Universal %}
+
+```yaml
+type: ExternalService
+mesh: default
+name: httpbin
+tags:
+  kuma.io/service: httpbin
+  kuma.io/protocol: http
+networking:
+  address: httpbin.org:443
+  tls:
+    enabled: true
+```
+
+{% endnavtab %}
+{% endnavtabs %}
+
+When applying policies, External Services are treated as normal Mesh Services, so we could configure MeshGlobalRatelimit like this:
+
+{% navtabs %}
+{% navtab Kubernetes %}
+
+```yaml
+apiVersion: kuma.io/v1alpha1
+kind: MeshGlobalRateLimit
+metadata:
+  name: httpbin-rate-limit
+  namespace: {{ site.mesh_namespace }}
+spec:
+  targetRef:
+    kind: MeshService
+    name: httpbin
+  from:
+    - targetRef:
+        kind: Mesh
+      default:
+        http:
+          requestRate:
+              num: 100
+              interval: 1s    
+```
+
+{% endnavtab %}
+{% navtab Universal %}
+
+```yaml
+type: MeshGlobalRateLimit
+name: httpbin-rate-limit
+mesh: default
+spec:
+  targetRef:
+    kind: MeshService
+    name: httpbin
+  from:
+    - targetRef:
+        kind: Mesh
+      default:
+        http:
+          requestRate:
+              num: 100
+              interval: 1s
+```
+
+{% endnavtab %}
+{% endnavtabs %}
+
+## Ratelimit Service
+
+{{site.mesh_product_name}} is using **Envoy Rate Limit service reference implementation**. Its source code with documentation can be found [here](https://github.com/envoyproxy/ratelimit).
+
+### Deployment
+
+On Universal, you need to take care of deploying Redis and ratelimit service on your own. On Kubernetes, you could use {{site.mesh_product_name}} Helm charts 
+to deploy ratelimit service, but you still need to deploy Redis on your own.
+
+### How to read limits configuration
+
+When you expose debug port on ratelimit service, you can get access to limits configuration, at `/rlconfig` path.
+Here is an example configuration that you will see something like this:
+
+```
+default.kuma.io/service_demo-app_kuma-demo_svc_5000: unit=SECOND requests_per_unit=10, shadow_mode: false
+```
+
+This config follows a pattern: 
+```
+mesh.descriptorKey_descriptorValue
+```
+Descriptor key will always be `kuma.io/service`, and descriptor value will be your Mesh Service name. 
+Therefore, we can deduct that this config will apply to `demo-app_kuma-demo_svc_5000` service in `default`
+mesh. The second part of the configuration is rather straight forward. We have information of request limit and unit to which 
+this limit will be applied. Shadow mode specifies if request should be limited after reaching the configured limit. If `shadow_mode` is set 
+to `true`, ratelimit service will not deny requests to your service, it will only update ratelimit service metrics.
```

Comment:
```suggestion
The descriptor key will always be `kuma.io/service`, and the descriptor value will be your mesh service name. 
Therefore, we can deduct that this configuration will apply to `demo-app_kuma-demo_svc_5000` service in the `default`
mesh. For the second part of the configuration, we have information about the request limit and unit to which 
this limit will be applied. Shadow mode specifies if request should be limited after reaching the configured limit. If `shadow_mode` is set 
to `true`, the ratelimit service will not deny requests to your service, it will only update the ratelimit service metrics.
```